### DISCUSSION TITLE
Handling multiple features at the same time

### DISCUSSION THREAD

**Discussion by jkimbo** [ejoffe/spr#250] [Q&A]
Hi all,

This looks like a great tool, I was just wondering how you handle working on multiple features/bug fixes at the same time? As far as I can see every commit on top of main gets treated as part of the feature “stack” but sometimes I’ll create a commit that doesn’t have to be part of the current feature work and could be released separately. Is there a way of me taking that commit and making a separate pr out of it?

Thanks!

**Comment by ejoffe** [comment]
You can create multiple branches, and each branch has its own separate stack of commits. 
For the most part I usually work in a single stack of commits on the master branch, but when I have a different feature that are completely unrelated (as you mention) I create a new branch and start a new stack of commits for that feature.
